typescript/test/testdata/texttospeech/src/v1/text_to_speech_client.ts.baseline

```diff
@@ -318,11 +318,11 @@ export class TextToSpeechClient {
  *
  * @param {Object} request
  *   The request object that will be sent.
- * @param {google.cloud.texttospeech.v1.SynthesisInput} request.input 
+ * @param {google.cloud.texttospeech.v1.SynthesisInput} [request.input] 
```

Comment:
Something is going wrong. This field must be marked as required in proto, why is it considered optional here?